packages/extensions/experience-platform-connector/src/handlers/shoppingCartPageView.js

```diff
@@ -0,0 +1,45 @@
+import { getCartTotal, getCurrency, getFormattedProducts } from '../utils';
+
+const canHandle = event => event.type === 'CART_PAGE_VIEW';
+
+const handle = (sdk, event) => {
+    const { payload } = event;
+
+    const { cart_id: id, products } = payload;
+
+    const cartContext = {
+        id: id,
```

Comment:
Optional suggestion for this and other instances in PR using the same key: value pair
```suggestion
        id,
```